docs/guides/writing_your_first_bot.mdx

```diff
@@ -61,52 +76,89 @@ Resolving dependencies...
 Changed 14 dependencies!
 ```
 
-</br>
-
 ### First code
 
 With everything set up, we can now start coding our bot, but let's start with something easy.
 This is simple bot which replies to `!ping` with `pong`.
-```dart
+
+```dart title="./bin/your_project_name.dart"
+import 'package:nyxx/nyxx.dart';
+
 void main() {
+  // highlight-start
   final bot = NyxxFactory.createNyxxWebsocket("TOKEN", GatewayIntents.allUnprivileged);
```

Comment:
Could you add the message content intent here now that is it privileged?

Reply:
Yup! My mistake..